Python/Setup/vsix.proj

```diff
@@ -26,6 +26,5 @@
   </Target>
 
   <Import Project="$(TargetsPath)\Common.Build.Traversal.targets" />
-  <Import Project="$(TargetsPath)\MicroBuild.targets" />
```

Comment:
MicroBuild.targets seems important. r you sure you want to get rid of that?

Reply:
This was removed because `Common.Build.targets` already imports it. It was a duplicate import 😄